pkg/event/targetlist.go

```diff
@@ -74,24 +74,15 @@ func (list *TargetList) Remove(targetids ...TargetID) <-chan TargetIDErr {
 	go func() {
```

Comment:
Can't understand the fix.

Reply:
It's a theoretical fix i.e avoid concurrent map read/write @balamurugana as indicated in the issue #6483

Reply:
where is concurrent read/write happening?

Reply:
```
   4 sep 18 17:10:14 xxxxx minio[845]: fatal error: concurrent map read and map write
   5 sep 18 17:10:14 xxxxx minio[845]: goroutine 529816617 [running]:
   6 sep 18 17:10:14 xxxxx minio[845]: runtime.throw(0x1486267, 0x21)
   7 sep 18 17:10:14 xxxxx minio[845]: /opt/go/src/runtime/panic.go:616 +0x81 fp=0xc42c046e88 sp=0xc42c046e68 pc=0x42ae41
   8 sep 18 17:10:14 xxxxx minio[845]: runtime.mapaccess2(0x1265860, 0xc420698c60, 0xc42c046f88, 0xc428a94480, 0xc423422a80)
   9 sep 18 17:10:14 xxxxx minio[845]: /opt/go/src/runtime/hashmap.go:409 +0x225 fp=0xc42c046ed0 sp=0xc42c046e88 pc=0x4077d5
  10 sep 18 17:10:14 xxxxx minio[845]: github.com/minio/minio/pkg/event.(*TargetList).Remove.func1(0xc42c557e60, 0xc427717420, 0x1, 0x1, 0xc4205bb120)
  11 sep 18 17:10:14 xxxxx minio[845]: /q/.q/sources/gopath/src/github.com/minio/minio/pkg/event/targetlist.go:79 +0x134 fp=0xc42c046fb8 sp=0xc42c046ed0 pc=0xb205c4
  12 sep 18 17:10:14 xxxxx minio[845]: runtime.goexit()
  13 sep 18 17:10:14 xxxxx minio[845]: /opt/go/src/runtime/asm_amd64.s:2361 +0x1 fp=0xc42c046fc0 sp=0xc42c046fb8 pc=0x458491
  14 sep 18 17:10:14 xxxxx minio[845]: created by github.com/minio/minio/pkg/event.(*TargetList).Remove
  15 sep 18 17:10:14 xxxxx minio[845]: /q/.q/sources/gopath/src/github.com/minio/minio/pkg/event/targetlist.go:74 +0xc3
```

So basically line number 74 and 79

Reply:
The fix does not change anything between line number 74 and 79.  It looks like `TargetList.Remove()` is called in parallel somewhere. that needs to be fixed.